src/backend/src/services/announcement.service.ts

```diff
@@ -0,0 +1,168 @@
+import { Announcement } from 'shared';
+import prisma from '../prisma/prisma';
+import { getAnnouncementQueryArgs } from '../prisma-query-args/announcements.query.args';
+import announcementTransformer from '../transformers/announcements.transformer';
+import { DeletedException, HttpException, NotFoundException } from '../utils/errors.utils';
+
+export default class AnnouncementService {
+  /**
+   * Creates an announcement that is sent to users
+   * this data is populated from slack events
+   * @param text slack message text
+   * @param usersReceivedIds users to send announcements to
+   * @param dateMessageSent date created of slack message
+   * @param senderName name of user who sent slack message
+   * @param slackEventId id of slack event (provided by slack api)
+   * @param slackChannelName name of channel message was sent in
+   * @param organizationId id of organization of users
+   * @returns the created announcement
+   */
+  static async createAnnouncement(
+    text: string,
+    usersReceivedIds: string[],
+    dateMessageSent: Date,
+    senderName: string,
+    slackEventId: string,
+    slackChannelName: string,
+    organizationId: string
+  ): Promise<Announcement> {
```

Comment:
we should probably validate that the usersReceivedIds are all valid ids. otherwise were gonna throw a weird error